.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,29 +1,21 @@
-## PR Summary
+## PR summary
 
-## PR Checklist
+## PR checklist
+<!-- Please mark any checkboxes that do not apply to this PR as [N/A].-->
 
-<!-- Please mark any checkboxes that do not apply to this PR as [N/A]. -->
-**Linked Issue**
-- [ ] Added "closes #0000" in the PR description to link it to the original issue.
-
-**Documentation and Tests**
-- [ ] Has pytest style unit tests (and `pytest` passes)
-- [ ] Documentation is sphinx and numpydoc compliant (the docs should [build](https://matplotlib.org/devel/documenting_mpl.html#building-the-docs) without error).
-- [ ] New plotting related features are documented with examples.
-
-**Release Notes**
-- [ ] New features are marked with a `.. versionadded::` directive in the docstring and documented in `doc/users/next_whats_new/`
-- [ ] API changes are marked with a `.. versionchanged::` directive in the docstring and documented in `doc/api/next_api_changes/`
-- [ ] Release notes conform with instructions in  `next_whats_new/README.rst` or `next_api_changes/README.rst`
+- [ ] Added "closes #0000" in the PR description to [link it](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) to the original issue.
+- [ ] Tested contribution using pytest style [tests](https://matplotlib.org/devdocs/devel/testing.html)
+- [ ] Documented contribution using [sphinx](https://matplotlib.org/devdocs/devel/documenting_mpl.html#writing-rest-pages) and [numpydoc](https://matplotlib.org/devdocs/devel/documenting_mpl.html#writing-docstrings)
```

Comment:
I'm not sure if this is as clear as the previous version. On the first read, it might look like `numpydoc` is a tool and not guidelines?

Reply:
Yeah, that sounds better I think. Tbh, I wouldn't mind not keeping it similar to the original version either, for this one

```markdown
Documentation is [sphinx rst](https://matplotlib.org/devdocs/devel/documenting_mpl.html#writing-rest-pages) and [numpydoc](https://matplotlib.org/devdocs/devel/documenting_mpl.html#writing-docstrings) compliant
```

Reply:
Do we need the technical Sphinx/numpydoc part at all? Suggestion:

Wrote documentation according to the [general documentation](…) and [docstring](…) guidelines.

Reply:
Motivation for the proposal: Abstraction level: I'd rather talk about the scopes instead of the technologies:
  - "Make sure you have written compliant prose documentation" and not "make sure you have used sphinx rst"
  - "Make sure you have written proper docstrings" and not "make sure you have used numpydoc"

In both cases, the points to watch out for is more than just the mentioned technology:

- **General documentation**: "sphinx ReST compliant" is not the central aspect of the documentation. A user does not have to specifically watch out for this. This is corrolary of the technology used (which the users either knows already or learns when looking into how to document). It's also partly checked automatically through the doc build.
What I find more important here are the conventions we impose (casing of section titles, referring to other code through the `py:obj` default role (i.e. no `:meth:`, `:class:` etc. prefixed) etc.
- **Docstrings**: We expand on numpydoc. Our docstring conventions are "numpydoc" + some extra rules.

Reply:
Maybe we don’t need the “pytest” any more either?  Possibly it was originally specified when we had a mixture of pytest and unittest (and nose?) in the existing tests, but wanted new ones to be pytest.

Given the other two doc points in the list tell you what kind of doc change are needed, I would maybe move the sphinx/numpydoc one to the end of the list and have something like “Any doc changes comply with…”

Reply:
I agree that "pytest style" is not really needed on this level. "Changes are covered by unit tests" is sufficient.

Reply:
Yeah, just a general statement about requirements should be good.